### PR TITLE
add /= doc

### PR DESCRIPTION
**Pull request by Jobhdez**

### CODE REVIEW

File: library/functions.lisp [hunk 1]
```diff
@@ -122,6 +122,7 @@
 
   (declare /= (Eq :a => :a -> :a -> Boolean))
   (define (/= a b)
+    "Returns TRUE if a is not equal to b."
```
Comment (stylewarning) [inline library/functions.lisp:125]: I like a more informal way of writing: "Is A not equal to B?"

Comment (stylewarning) [reply library/functions.lisp:125]: (Regardless of how you decide, it should be written `\`a\`` or `A`; and `True` not `TRUE` as is conventional for ADT constructors.)

**Review by stylewarning — CHANGES_REQUESTED** [review]

**Review by eliaslfox — APPROVED** [review]

**Review by stylewarning — APPROVED** [review]

**Comment by stylewarning** [comment]
@Jobhdez can you please squash your commits?

**Comment by Jobhdez** [comment]
> @Jobhdez can you please squash your commits?

ok I did it

**Pull request merged**